src/routes/Review/components/ReviewTable/index.jsx

```diff
@@ -1,174 +1,39 @@
-import { Link } from "@reach/router";
-import classNames from "classnames";
-import PageDivider from "components/PageDivider";
-import { ProgressLevels } from "constants/";
-import React, { useState } from "react";
-import _ from "lodash";
-import ArrowIcon from "../../../../assets/images/icon-arrow.svg";
-import "./styles.module.scss";
+import React from "react";
+import { WorkType } from "../../../../../src-ts/lib";
+import { WorkDetailDetailsPane } from "../../../../../src-ts/tools/work/work-detail-details/work-detail-details-pane";
```

Comment:
We should not be importing anything from the this deep into the tools. Let's add this to the import from /src-ts

Reply:
Done